samples/web/content/testrtc/js/camresolutionstest.js

```diff
@@ -7,44 +7,96 @@
  */
 'use strict';
 
-/* This test tries calling getUserMedia() with each resolution from the list
- * below. Each gUM() call triggers a success or a fail callback; we report
- * ok/nok and schedule another gUM() with the next resolution until the list
- * is exhausted. Some resolutions are mandatory and make the test fail if not
- * supported.
+ /*
+ * Analyze performance for X resolution uses getStats, canvas and the video
```

Comment:
Move this paragraph next to the code that add the "Analyze performance for X" tests. Also quote the test name so it clear it is a name and not part of the sentence.